docs/source/conf.py

```diff
@@ -16,7 +16,6 @@
 extensions = [
     "myst_parser",
     "sphinx_copybutton",
```

Comment:
```suggestion
    "sphinx_copybutton",
    "jupyterhub_sphinx_theme",
```

Let's add this back, the bug of not respecting `html_logo` should be fixed instead, as proposed via https://github.com/jupyterhub/jupyterhub-sphinx-theme/pull/13